scenario/consistency_checker.py

```diff
@@ -560,6 +560,7 @@ def check_containers_consistency(
     meta_containers = list(map(normalize_name, meta.get("containers", {})))
     state_containers = [normalize_name(c.name) for c in state.containers]
     all_notices = {notice.id for c in state.containers for notice in c.notices}
+    all_checks = {check.name for c in state.containers for check in c.check_infos}
```

Comment:
I was going to ask: what if two containers share the same check name?
But then I saw that this variable is only used for one specific validation below.
I guess it's nothing, it was just surprising when I read it.